src/renderer/i18n/nl.js

```diff
@@ -0,0 +1,483 @@
+    experimentalFirmware: "Chrysalis {{version}} experimenteel",
+    experimentalFirmwareDescription: "Experimenteel, met meer plugins aangezet",
+    selected: "Geselecteerde firmware",
+    custom: "Custom firmware",
+    description: `Updaten of "flashen" van de firmware van je toetsenbord is hoe we het nieuwe kunstjes leren. Chrysalis zal een nieuwe versie van de firmware van je toestenbord installeren. Dit zorgt voor ondersteuning van een aangepaste toetsen layout, naast andere features. Indien je hiervoor al de firmware van je toetsenbord hebt aangepast, zal dit de aangepaste firmware overschrijven. Je kan de broncode van de firmware die Chrysalis aan het installeren is hier vinden:`,
+    postUpload: `Wanneer de upload gedaan is, zal Chrysalis je terugbrengen naar het selectiescherm voor je toetsenbord.`,
+  },
+  "focus-not-detected": {
+    title: "Welkom bij Chrysalis",
+    contents: `Chrysalis herkent je toetsenbord maar moet de firmware updaten voordat je verder kan doen.`,
+    gotoUpdate: "Update Firmware",
+    reconnect: "Opnieuw verbinden",
+    reconnectDescription: `Er is een kans dat we de mogelijkheden van je toetsenbord fout gedetecteerd hebben, of dat je toetsenbord nog aan het opstarten was terwijl we verbinding gemaakt hebben. In dat geval kan je proberen om te klikken op de "{{buttonName}}" knop om opnieuw te proberen verbinden en nogmaals te zoeken naar de nodige features. Opnieuw verbinden is normaal enkel nuttig als je zeker bent dat er een tijdelijk probleem was tijdens vorige pogingen dat in de tussentijd is opgelost.`,
+  },
+  systeminfo: {
+    title: "Rapporteer een probleem",
+    intro: `Als je een probleem hebt met Chrysalis kan het ontwikkelteam je vragen om hen wat debug informatie door te sturen. Chrysalis kan je helpen om alles dat je eventueel moet delen te verzamelen en in te pakken. Dit omvat error logs en informatie over het besturingssysteem, verbonden toestellen en Chrysalis zelf. Toetsenbord informatie omvat info over de huidige firmware, de toets layouts en de LED configuratie van je toetsenbord.`,
+    bugTracker: `Je kan een lijst van gekende problemen vinden op GitHub (of een bug rapporteren)`,
+    privacyNote: `We doen ons best om geen gevoelige informatie te includeren in deze bundel, maar we raden je aan om erdoor te lezen alvorens hem te delen.`,
+    createBundle: "Maak bundel aan",
+    viewBundle: "Bekijk bundel",
+    saveBundle: "Bundel opslaan",
+    dialog: {
+      title: "Chrysalis Debug Bundel opslaan",
+      bundleFiles: "Debug bundel bestanden",
+    },
+    bundleSaved: `Chrysalis debug bundel opgeslaan.`,
+  },
+  devices: {
+    Dygma: {
+      Raise: {
+        updateInstructions: `Om de firmware te updaten heeft je toestenbord nood aan een speciale reset. Wanneer je het licht op de Neuron ziet uitgaan, blijf dan de Escape toets ingedrukt houden. Het licht van de Neuron zou een blauw pulserend patroon moeten opstarten.`,
```

Comment:
Simplify `nood hebben aan` to `vereisen`.

```suggestion
        updateInstructions: `Om de firmware te updaten vereist je toestenbord een speciale reset. Wanneer je het licht op de Neuron ziet uitgaan, blijf dan de Escape toets ingedrukt houden. Het licht van de Neuron zou een blauw pulserend patroon moeten opstarten.`,
```